Also, Falling `Backend tests`
```
FAIL: test_skill_ids_validation (core.domain.collection_domain_test.CollectionDomainUnitTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/oppia/oppia/core/domain/collection_domain_test.py", line 202, in test_skill_ids_validation
    'Expected skill ID to have length at least 6, received a')
  File "/home/travis/build/oppia/oppia/core/domain/collection_domain_test.py", line 74, in _assert_validation_error
    self.collection.validate()
AssertionError: "Expected skill ID to have length at least 6, received a" does not match "Expected skill ID to begin with 'skill', received a"
```

--- a/core/domain/collection_domain.py
+++ b/core/domain/collection_domain.py
@@ -391,7 +391,7 @@ class CollectionSkill(object):
 
     @classmethod
     def from_dict(cls, skill_id, skill_dict):
-        """Returns the CollectionSkill object corresponding to the given 
+        """Returns the CollectionSkill object corresponding to the given
         skill_id and skill_dict.
 
         Args:

--- a/core/domain/collection_domain_test.py
+++ b/core/domain/collection_domain_test.py
@@ -198,9 +198,6 @@ class CollectionDomainUnitTests(test_utils.GenericTestBase):
                 'question_ids': []
             })}
 
-        self._assert_validation_error(
-            'Expected skill ID to have length at least 6, received a')
-
         self.collection.skills = {
             'abcdef': collection_domain.CollectionSkill.from_dict(
                 'test_skill', {

